src/drivers/simulink/simulink.py

```diff
@@ -0,0 +1,172 @@
+# Simumatik Gateway - Simumatik 3rd party integration tool
+# Copyright (C) 2021 Simumatik AB
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+from multiprocessing import Pipe
+from typing import Optional
+
+from ..driver import driver, VariableQuality
+import os
+import sys
+import winreg
+
+MATLAB_SDK_FOUND = False
+
+def add_matlab_path():
+    '''
+    Tries to append the path of the matlab engine.
+    If successfull, MATLAB_SDK_FOUND is set to True. Otherwise an error is raised.
+    '''
+    if os.name == 'nt': # Only try on windows
+        reg = winreg.ConnectRegistry(None, winreg.HKEY_LOCAL_MACHINE)
+
+        # Get registry key to the latest version of MATLAB installed
+        key = winreg.OpenKey(reg, "SOFTWARE\\MathWorks\\MATLAB")
+        matlab_version = winreg.EnumKey(key, 0)
+        key = winreg.OpenKey(reg, "SOFTWARE\\MathWorks\\MATLAB\\" + matlab_version)
+        
+        # Path to python engine relative to MATLABROOT
+        matlab_path = winreg.QueryValueEx(key, "MATLABROOT")[0] + "\\extern\\engines\\python"
+        assert os.path.isdir(matlab_path + "\\build\\lib"), "Matlab engine not built"
```

Comment:
Just as a side note, be careful when using asserts, as it could make the driver panic if used outside a try block.